tests/unit/test_winAPI/test_powerTracking.py

```diff
@@ -0,0 +1,143 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+from typing import cast
+import unittest
+from unittest.mock import MagicMock
+
+from winAPI.powerTracking import (
+	BATTERY_LIFE_TIME_UNKNOWN,
+	BatteryFlag,
+	PowerState,
+	SystemPowerStatus,
+	_getSpeechForBatteryStatus,
+)
+
+
+class Test_GetSpeechForBatteryStatus(unittest.TestCase):
+	def setUp(self) -> None:
+		self.testPowerStatus = cast(
+			SystemPowerStatus,
+			MagicMock(SystemPowerStatus())
+		)
+
+	def test_unknownPowerStatus_failedFetch(self):
+		actualSpeech = _getSpeechForBatteryStatus(
+			systemPowerStatus=None,
+			onlyReportIfStatusChanged=False,
+			oldPowerState=PowerState.UNKNOWN,
+		)
+		self.assertListEqual(
+			["Unknown power status"],
+			actualSpeech,
+		)
+
+	def test_unknownPowerStatus_fetchSuccessful(self):
+		self.testPowerStatus.BatteryFlag = BatteryFlag.UNKNOWN
+		actualSpeech = _getSpeechForBatteryStatus(
+			systemPowerStatus=self.testPowerStatus,
+			onlyReportIfStatusChanged=False,
+			oldPowerState=PowerState.UNKNOWN,
+		)
+		self.assertListEqual(
+			["Unknown power status"],
+			actualSpeech,
+		)
+
+	def test_noSystemBattery(self):
+		self.testPowerStatus.BatteryFlag = 0 ^ BatteryFlag.NO_SYSTEM_BATTERY
+		actualSpeech = _getSpeechForBatteryStatus(
+			systemPowerStatus=self.testPowerStatus,
+			onlyReportIfStatusChanged=False,
+			oldPowerState=PowerState.UNKNOWN,
+		)
+		self.assertListEqual(

```

Comment:
The documentation for `assertListEqual` states that it should not be used directly, and `assertEqual` automatically uses it for lists.

Reply:
In the paragraph for [type specific methods](https://docs.python.org/3.7/library/unittest.html#type-specific-methods) there is the following remark:
> The list of type-specific methods automatically used by assertEqual() are summarized in the following table. Note that it’s usually not necessary to invoke these methods directly.

Reply:
I don't think that is the same as discouraging direct usage

Reply:
The main reason why I flagged this is that `assertListEqual` is almost never used since, as documented in the Python documentation, `assertEqual` uses it automatically for lists. For someone reading this code it is unclear why you've decided to use it directly.